### ISSUE TITLE
npm-link-better --copy does not copy node_modules

### ISSUE THREAD

**Issue by bodograumann** [laggingreflex/npm-link-better#6]
When running `nlc`, it copies the whole directory, except `node_modules` and `.git`.
I can understand leaving out `.git`, but why not copy `node_modules`?
This way all the sub-dependencies are missing.

**Comment by laggingreflex** [comment]
For me personally this was desirable since usually my linked modules's dependencies already exist in project's dependency since `npm i` flattens them while installing. 

But yeah, a better solution is to let the user customize include/exclude pattern.

**Comment by bodograumann** [comment]
I agree that this might often be the case, but currently I was building a completely new package :-)
Also the sub-dependencies might change.
Not sure if there could be problems if a dependency gets duplicated.
